### PR TITLE
suffix for time step selector

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[BC in Zeitleiste](https://trello.com/c/UROqngKG/1500-bc-in-zeitleiste)

File: projects/core/src/lib/time/time-step-selector/time-step-selector.component.ts [hunk 1]
```diff
@@ -33,6 +38,15 @@ export class TimeStepSelectorComponent implements OnChanges {
             map(([time, format]) => {
                 if (this.timeSteps) {
                     this.currentTimeIndex = this.timeSteps.findIndex((t) => time.isSame(t));
+
+                    if (this.useCeFormat) {
+                        if (time.start.year() < 0) {
+                            format = format.replace('YYYY', 'YYYY [BCE]');
+                        } else {
+                            format = format.replace('YYYY', 'YYYY [CE]');
```
Comment (michaelmattig) [inline projects/core/src/lib/time/time-step-selector/time-step-selector.component.ts:46]: should be AD, if they want BC.

File: projects/core/src/lib/time/time-step-selector/time-step-selector.component.ts [hunk 2]
```diff
@@ -33,6 +38,15 @@ export class TimeStepSelectorComponent implements OnChanges {
             map(([time, format]) => {
                 if (this.timeSteps) {
                     this.currentTimeIndex = this.timeSteps.findIndex((t) => time.isSame(t));
+
+                    if (this.useCeFormat) {
+                        if (time.start.year() < 0) {
+                            format = format.replace('YYYY', 'YYYY [BCE]');
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply projects/core/src/lib/time/time-step-selector/time-step-selector.component.ts]: > * why the brackets?

Didn't you execute the UI? It is some string that is not a modifier.

> 
>     * why BCE? Didn't they ask for BC (which is equivalent but just alternativate notation)?


[2333641](https://github.com/geo-engine/geoengine-ui/pull/681/commits/2333641aa6d94454f4336074999fd4fb3d06603b)

File: projects/core/src/lib/time/time-step-selector/time-step-selector.component.ts (hunk 1 above)
Comment (ChristianBeilschmidt) [reply projects/core/src/lib/time/time-step-selector/time-step-selector.component.ts]: [2333641](https://github.com/geo-engine/geoengine-ui/pull/681/commits/2333641aa6d94454f4336074999fd4fb3d06603b)

**Review by michaelmattig — APPROVED** [review]

**Pull request merged**